test-requirements.txt

```diff
@@ -2,7 +2,7 @@ mock
 pur
 # pytest==3.1.0 issues with pytest-verbose-parametrize
 pytest==3.0.7
-pytest-cov
+pytest-cov==2.6.0
```

Comment:
I still want conditionals I think.

Reply:
env markers. anyway, let's merge it as is for now. but for future I'd like to upgrade pytest, there's even a new major version out there.

Reply:
I'm not sure what env marker would be relevant in that context, hence my question. There is no dependency on a specific version of python and I can't think of any other env marker.